python/paddle/nn/layer/common.py

```diff
@@ -248,3 +251,93 @@ def forward(self, input):
             data_format=self.data_format)
 
         return out
+
+
+class Pad2D(layers.Layer):
+    """
+        :alias_main: paddle.nn.Pad2D
+        :alias: paddle.nn.Pad2D,paddle.nn.layer.Pad2D,paddle.nn.layer.common.Pad2D
+
+    This interface is used to construct a callable object of the ``Pad2D``  class.
+    The Pad2D layer pad 2-d images according to 'paddings' and 'mode'.
```

Comment:
pad 2-d images ->  pads the input tensor boundaries

Reply:
done